src/Datadog.Trace/TracingProcessManager.cs

```diff
@@ -53,19 +53,19 @@ public static void Initialize()
 
                 if (DomainMetadata.ShouldAvoidAppDomain())
                 {
-                    Log.Information("Skipping process manager initialization for AppDomain: {0}", DomainMetadata.AppDomainName);
+                    Log.Debug("Skipping process manager initialization for AppDomain: {AppDomain}", DomainMetadata.AppDomainName);
```

Comment:
@colin-higgins, master of all things AAS: what do you think of changing this message to debug level? (not emitted by default anymore)

Reply:
This is valuable information that gives important context on the environment in support cases.

Reply:
Do we want to keep writing it by default or change it to only write it in debug mode?

Reply:
Reverted this change back to Information